core/src/main/java/org/apache/iceberg/ManifestEntriesTable.java

```diff
@@ -112,28 +113,36 @@ protected long targetSplitSize(TableOperations ops) {
       ResidualEvaluator residuals = ResidualEvaluator.unpartitioned(filter);
 
       return CloseableIterable.transform(manifests, manifest ->
-          new ManifestReadTask(ops.io(), manifest, fileSchema, schemaString, specString, residuals));
+          new ManifestReadTask(ops.io(), manifest, fileSchema, schemaString, specString, residuals, ops.current()
+              .specsById()));
```

Comment:
Nit: I think it's usually better to wrap a line at the start of an argument instead of at a method call in an argument, so `\nops.current().specsById()` instead of `ops.current()\n.specsById()`. Otherwise, it looks like you may be calling `specsById` on `ManifestReadTask`.

Reply:
Sounds good to me, My preferred is new lines for every arg every time :) But I think this is reasonable too